lib/helpers/digital_ocean_helper.rb

```diff
@@ -2,69 +2,67 @@
 
 # Class represent rake helper methods
 class DigitalOceanHelper
-  class << self
-    attr_accessor :init_api
+  attr_accessor :init_api
```

Comment:
`:do_api` I think is better

And maybe make this not `attr_accessor`, but `att_reader` and add `@do_api = OnlyofficeDigitaloceanWrapper::DigitalOceanWrapper.new` (or something like this) in constructor?

Reply:
I think we need use atr_accessor but

Reply:
and @do_api really lucky name

Reply:
No, https://github.com/askonev/onlyoffice-droplets-starter/blob/b9703257b668400cbf9f544e2c41b120319b8846/lib/helpers/digital_ocean_helper.rb#L59

This case will work fine with attr_reader, since it's not writing into value of `@init_api` it just calling it and using it, but not changing value (attr_writer is used to change value of variable itself)

Reply:
> `:do_api` I think is better
> 
> And maybe make this not `attr_accessor`, but `att_reader` and add `@do_api = OnlyofficeDigitaloceanWrapper::DigitalOceanWrapper.new` (or something like this) in constructor?

If we use attr_reader `NoMethodError: undefined method do_api= for #<DigitalOceanHelper:0x0000562d1e8c0338>`

In [this part](https://github.com/askonev/onlyoffice-droplets-starter/blob/b9703257b668400cbf9f544e2c41b120319b8846/Rakefile#L6)  of the code, we are initializing the DigitalOceanHelper class. Those need to be granted write access to the *do_api* variable using the *attr_writer* method

Reply:
Got it, agree about this part